### PR TITLE
Japanese translation added

### PR DESCRIPTION
**Pull request by funax**
I tested on my self-hosted Puter.
Please ignore the change made to "global.js"

### CODE REVIEW

**Comment by jelveh** [comment]
Thank you very much for your contribution! Could you please look into the [CLA](https://github.com/HeyPuter/puter/pull/392#issuecomment-2105598120) before I can merge this?

File: src/globals.js [hunk 1]
```diff
@@ -228,4 +228,4 @@ window.desktop_item_positions = {};
 window.reset_item_positions = true; // The variable decides if the item positions should be reset when the user enabled auto arrange
 
 // default language
-window.locale = 'en';
\ No newline at end of file
+window.locale = 'ja';
```
Comment (AtkinsSJ) [inline src/globals.js]: Please don't change the default language

File: src/images/wallpaper.webp
Comment (AtkinsSJ) [inline src/images/wallpaper.webp:1]: What is this wallpaper file doing here?

**Review by AtkinsSJ — CHANGES_REQUESTED** [review]

**Comment by jelveh** [comment]
@musaden could you look into the CLA as well?

**Pull request closed without merging**